lab-exercise-1/ChalisaFeature.java

```diff
@@ -0,0 +1,5 @@
+public class ChalisaFeature {
+    public void feature() {
+  // this is a new feature
+}
```

Comment:
This content is not well-written. Please improve it.

Reply:
This content is not well-written. please improve it.

Reply:
Pls improve I think can better.